returnn/tf/network.py

```diff
@@ -1337,6 +1337,7 @@ def _map_layer_dict_value(v):
 
     def _make_layer(layer_cls, layer_dict, map_opts=True):
       """
+      Creates the flattened layer
```

Comment:
There should be an empty line after the description before the params (when the description is non-empty).

(Here and for all further docstrings as well.)